modules/App.js

```diff
@@ -1,53 +1,63 @@
-import React from 'react'
-import { Link } from 'react-router'
-import { IndexLink } from 'react-router'
+import React from 'react';
+import { Link } from 'react-router';
+import { IndexLink } from 'react-router';
 import { Navbar } from 'react-bootstrap';
 import { connect } from 'react-redux';
-import SideBar from './SideBar'
-
-import QueryBox from './QueryBox'
+import SideBar from './SideBar';
+import PropTypes from 'prop-types';
+import QueryBox from './QueryBox';
 
 class App extends React.Component {
-    constructor(props) {
-        super(props);
-        this.state = {
-            openSideBar: false,
-        };
-    }
+  constructor(props) {
+    super(props);
+    this.state = {
+      openSideBar: false,
+    };
+  }
 
-    render() {
-        return (
-            <div className='app'>
-                <Navbar fixedTop={true} fluid={true}>
-                    <IndexLink to='/'>
-                        <img width='20' src='/images/logo.png' />
-                        <span className='brand-title'>Bazaar</span>
-                    </IndexLink>
-                    {
-                        this.props.location.pathname === '/'
-                        && this.props.isAuthenticated
-                        && <QueryBox />
-                    }
-                    <button onClick={() => this.setState({openSideBar: !this.state.openSideBar})} id='menu-toggle' type='button' className='navbar-toggle'>
-                        <span className='sr-only'>Toggle navigation</span>
-                        <span className='icon-bar'></span>
-                        <span className='icon-bar'></span>
-                        <span className='icon-bar'></span>
-                    </button>
-                </Navbar>
-                <SideBar toggled={this.state.openSideBar} />
-                {this.props.children}
-            </div>
-        )
-    }
+  render() {
```

Comment:
So is the linter configured to use two spaces (rather than 4) for code block indentation?